### PR TITLE
LPS-133404 | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-133404

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsConfiguration.testcase [hunk 1]
```diff
@@ -751,6 +751,120 @@ definition {
...
+		task ("Add a new fragment") {
+			FragmentsAdmin.addFragment(fragmentName = "Fragment Name");
+
+			var htmlSnippet = '''<h2 data-lfr-editable-id="title1" data-lfr-editable-type="rich-text">''';
+
+			FragmentsEditor.typeHtmlEditor(content = "${htmlSnippet}");
+
+			FragmentsEditor.publish();
+
+			VerifyElementPresent(locator1 = "Message#SUCCESS");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsConfiguration.testcase:766]: Use json to add a fragment

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsConfiguration.testcase [hunk 2]
```diff
@@ -751,6 +751,120 @@ definition {
...
+			var htmlSnippet = '''
+			<h2 data-lfr-editable-id="title1" data-lfr-editable-type="rich-text">
+			<h2 data-lfr-editable-id="title2" data-lfr-editable-type="rich-text">
+			''';
+
+			FragmentsEditor.typeHtmlEditor(content = "${htmlSnippet}");
+
+			FragmentsEditor.publish();
+
+			VerifyElementPresent(locator1 = "Message#SUCCESS");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsConfiguration.testcase:825]: Redundant step

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsConfiguration.testcase [hunk 3]
```diff
@@ -751,6 +751,120 @@ definition {
...
+				entry = "Fragment Name",
+				item = "Edit");
+
+			var htmlSnippet = '''
+			<h2 data-lfr-editable-id="title1" data-lfr-editable-type="rich-text">
+			<h2 data-lfr-editable-id="title2" data-lfr-editable-type="rich-text">
+			''';
+
+			FragmentsEditor.typeHtmlEditor(content = "${htmlSnippet}");
+
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsConfiguration.testcase:822]: Add step to wait for draft saved

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsConfiguration.testcase [hunk 4]
```diff
@@ -751,6 +751,120 @@ definition {
...
+				siteName = "Test Site Name");
+
+			for (var n : list "1,2") {
+				AssertVisible(
+					id = "title${n}",
+					key_position = "1",
+					key_text = "Example Rich Text",
+					locator1 = "Fragment#CONTRIBUTED_FRAGMENT_EDITABLE_FIELD_AT_VIEW_MODE");
+			}
+		}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsConfiguration.testcase:865]: Reorder the two tasks

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsConfiguration.testcase [hunk 5]
```diff
@@ -751,6 +751,120 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-118241. The propagate changes of the fragment should be applied to the page."
+	@priority = "4"
+	test ViewPropagatedChangesOfFragmentConfiguration {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsConfiguration.testcase:756]: PropagateChangesOfFragmentToContentPage

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsConfiguration.testcase [hunk 6]
```diff
@@ -751,6 +751,120 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-118241. The propagate changes of the fragment should be applied to the page."
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsConfiguration.testcase:754]: The user could propagate the changes of fragment to content page

**Comment by Tim-Cao** [comment]
Hi @ozysouza, could you move the case to FragmentsUI.testcase and update the case according to comments? Thanks

**Comment by ozysouza** [comment]
> Hi @ozysouza, could you move the case to FragmentsUI.testcase and update the case according to comments? Thanks

Hi @Tim-Cao PR Updated following your comments.

**Comment by Tim-Cao** [comment]
Hi @ozysouza, see changes https://github.com/liferay-echo/liferay-portal/compare/d626be691f103c84f920a84597b2176fc492ebde...5e5c9e1497ec2d71016a22df85743a92e2ee6417

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**